bin/setup

```diff
@@ -1,32 +1,19 @@
-#!/usr/bin/env ruby
```

Comment:
You need this line to be able to run this file as an executable

Reply:
👍